specs/latest/2.0/index.html

```diff
@@ -3326,6 +3362,48 @@ <h2 class="no-toc">Table of contents</h2>
         this limitation in WebGL 2 for all platforms.
     </div>
 
+    <h3><a name="PIXEL_STORE_PARAM_CONSTRAINTS">Pixel store parameter constraints</a></h3>
+
+    <p>
+        Define:
+        <ul>
+          <li>DataStoreWidth := <code>ROW_LENGTH</code> ? <code>ROW_LENGTH</code> : <em>width</em></li>
+          <li>DataStoreHeight := <code>IMAGE_HEIGHT</code> ? <code>IMAGE_HEIGHT</code> : <em>height</em></li>
+        </ul>
+    </p>
+
+    <p>
+        If <code>PACK_SKIP_PIXELS</code> + <em>width</em> > DataStoreWidth, <code>readPixels</code> generates an
```

Comment:
Does this HTML render correctly? I've found it necessary to use the "&lt;" and "&gt;" entities in the past.